package.json

```
@@ -1,6 +1,7 @@
 {
   "name": "nextchat-web",
   "version": "1.0.0-alpha",
+  "license": "GPL-3.0-or-later",
```

No creo que sea bueno dejar el `-or-later` porque significa que si sale una nueva versión y no se adapta a lo que necesitamos, pues se estaría cumpliendo con ella y nosotros sin saber.

```suggestion
  "license": "GPL-3.0",
```